### DISCUSSION TITLE
Can I import all known moons of Jupiter to Stellarium?

### DISCUSSION THREAD

**Discussion by JeffSG** [Stellarium/stellarium#1953] [Q&A] [answered]
Hi all, is it possible to add all the known moons of Jupiter to Stellarium?
If yes, may I know how to?
Thanks,
Jeff

**Comment by alex-w** [comment, marked as answer]
See Appendix D.2 in Stellarium User Guide + https://ssd.jpl.nasa.gov/horizons/

**Reply by JeffSG** [reply]
Thank you Alex for answering. I found some configuration items from the Horizons page. But I could not find the values of orbit_Ascendingnode, orbit_Epoch, orbit_LongOfPericenter, and orbit_MeanLongitude. 
Below is a sample result from the Horizons App for Elara, which is a moon of Jupiter. I guess "2454619.500000000 = A.D. 2008-Jun-02 00:00:00.0000 TDB " means the epoch. But how about the rest?
*******************************************************************************
 Revised: Mar 07, 2021            Elara   / (Jupiter)                       507

 SATELLITE PHYSICAL PROPERTIES:
  Radius (km)             = 40 +- 10        Density (g cm^-3)   =              
  Mass (10^20 kg )        =                 Geometric Albedo    =   0.03
                                            V(1,0)              = +10.07

 SATELLITE ORBITAL DATA:
  Semi-major axis, a (km) = 11737 (10^3)    Orbital period      = 259.6528 d
  Eccentricity, e         =   0.207         Rotational period   = 0.5 d 
  Inclination, i  (deg)   =  28             Local invar ref. pln= Sun
*******************************************************************************


*******************************************************************************
Ephemeris / API_USER Tue Oct  5 23:39:40 2021 Pasadena, USA      / Horizons
*******************************************************************************
Target body name: Elara (507)                     {source: jup344_merged}
Center body name: Earth (399)                     {source: DE441}
Center-site name: (user defined site below)
*******************************************************************************
Start time      : A.D. 2008-Jun-02 00:00:00.0000 TDB
Stop  time      : A.D. 2008-Jun-03 00:00:00.0000 TDB
Step-size       : 1440 minutes
*******************************************************************************
Center geodetic : 116.383300,39.9166000,0.0000000 {E-lon(deg),Lat(deg),Alt(km)}
Center cylindric: 116.383300,4898.65479,4070.8875 {E-lon(deg),Dxy(km),Dz(km)}
Center pole/equ : ITRF93 (high precision)         {East-longitude positive}
Center radii    : 6378.1 x 6378.1 x 6356.8 km     {Equator, meridian, pole}    
Output units    : KM-S
Output type     : GEOMETRIC cartesian states
Output format   : 3 (position, velocity, LT, range, range-rate)
EOP file        : eop.211004.p211228
EOP coverage    : DATA-BASED 1962-JAN-20 TO 2021-OCT-04. PREDICTS-> 2021-DEC-27
Reference frame : Ecliptic of J2000.0
*******************************************************************************
JDTDB
   X     Y     Z
   VX    VY    VZ
   LT    RG    RR
*******************************************************************************
$$SOE
2454619.500000000 = A.D. 2008-Jun-02 00:00:00.0000 TDB 
 X = 2.288977922240822E+08 Y =-6.022318452342229E+08 Z = 3.472453927343100E+06
 VX=-1.604139418203667E+01 VY= 1.012061264925846E+01 VZ= 6.933986867683952E-01
 LT= 2.149067943882219E+03 RG= 6.442743613054563E+08 RR=-1.515563513710226E+01
2454620.500000000 = A.D. 2008-Jun-03 00:00:00.0000 TDB 
 X = 2.275035758107346E+08 Y =-6.013507124526742E+08 Z = 3.519060556451380E+06
 VX=-1.613965843756011E+01 VY= 9.627298252569819E+00 VZ= 6.663731896214928E-01
 LT= 2.144671718567995E+03 RG= 6.429564061125836E+08 RR=-1.471152254266880E+01
$$EOE
*******************************************************************************

Thanks,
Jeff

**Reply by alex-w** [reply]
You need to obtain orbital elements, not a vector table.

P.S. Probably this link will be easiest: https://ssd.jpl.nasa.gov/sats/elem/

**Reply by JeffSG** [reply]
Got it. Appreciate your help Alex :)
